extension/src/prototypes/view/ui-containers/AppHeader.tsx

```diff
@@ -38,6 +59,13 @@ export const AppHeader: FC = () => {
       <IconButton>
         <PaperPlaneTilt onClick={() => setShowShareModal(true)} />
       </IconButton>
+      {isMobile && arSharedLayers && (
```

Comment:
What is arSharedLayers for?

```suggestion
      {isMobile && (
```